### DISCUSSION TITLE
Remove data from feed for projects I no longer watch

### DISCUSSION THREAD

**Discussion by jhubsharp** [community/community#20104] [Other Feature Feedback, Questions, & Ideas]
Is it possible to clean up the "Following" feed to remove data about projects I no longer watch or follow? I'm not seeing new data come in, but I'd like to get rid of all the historical data since I no longer follow the repository.

**Comment by Zoltus** [comment]
> Is it possible to clean up the "Following" feed to remove data about projects I no longer watch or follow? I'm not seeing new data come in, but I'd like to get rid of all the historical data since I no longer follow the repository.

Click the 3 dots on the corner and it should tell you why you are seeing those feeds.
![oOeCAh](https://user-images.githubusercontent.com/23402980/177573738-80687f31-b034-4ea4-84e5-2e3b55ede052.png)

If that doesnt work you could check incase if you are following the repo or user.

Third option would be bad but you could block the user. (And mayby try to unblock right after)?
Or keep it as blocked.

I hope u can fix ur feed, sounds annoying.

**Reply by jhubsharp** [reply]
Sadly, that doesn't seem to be the answer. Here's what I'm trying to clean up:
![image](https://user-images.githubusercontent.com/1591090/177574950-fa511d19-34dc-4895-ba93-13bd10965440.png)

For those two entries (and many more) in my "Following", they're old projects I'm no longer interested in seeing. I've checked that I'm not following the author or the repos (in fact, several of the repos I can't even access anymore - I get a 404 if I try). 

Certainly nothing critical but it would be nice to get old history out so I am only seeing past updates for the projects I still track.

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Comment by jhubsharp** [comment]
@jdevstatic That doesn't address the issue at hand in that I have a number of old updates in my Following feed that I no longer want to see.

**Comment by mikekavouras** [comment]
@jhubsharp unfortunately the 'Following' feed does not update retroactively, so there's no way to remove specific events. Staff does have the ability to completely clear your `Following` feed. Let me know if that's a desirable option and I can make it happen. 

Alternatively you can use the 'For you' feed with filters. Hope this helps!
